src/cmd/src/cli/export.rs

```diff
@@ -270,7 +300,7 @@ impl Export {
             })
             .count();
 
-        info!("success {success}/{db_count} jobs");
+        info!("Success {success}/{db_count} jobs");
```

Comment:
👍  Can we add some timing info to the final logs? Such as `Success {success}/{db_count} jobs, Cost {cost} seconds.` etc.